themes/default/content/blog/mlops-the-ai-challenge-is-cloud-not-code/index.md

```diff
@@ -0,0 +1,189 @@
+---
+date: 2023-08-30
+title: "The Real AI Challenge is Cloud, not Code!"
+allow_long_title: true
+meta_desc: "Accelerating AI/ML innovation with infrastructure as code through a hands-on journey to deploy your own chatbot"
+meta_image: "meta.png"
+authors:
+    - kat-morgan
+tags:
+    - ai
+    - ml
+    - iac
+    - llm
+    - vllm
+    - mlops
+    - azure
+    - llama
+    - llama2
+    - python
+---
+
+The AI industry is stealing the show as tech's goldrush of the '20s. Just looking at ChatGPT's [record setting user growth](https://www.reuters.com/technology/chatgpt-sets-record-fastest-growing-user-base-analyst-note-2023-02-01/), and rapid 3rd party integration [by top brands](https://www.forbes.com/sites/bernardmarr/2023/05/30/10-amazing-real-world-examples-of-how-companies-are-using-chatgpt-in-2023/?sh=ed1c90f14418), it is not surprising the hype suggests this is the beginning of a major digital transformation.
+
+When it comes to taking advantage of AI/ML in your own organization, realizing the benefits comes with major obstacles, not least of which is the almost complete lack of well practiced platform engineering architectures, industry consensus, and operational knowledge. Which is why it is easy to say, *The most difficult challenge that we face with AI today is a problem of cloud orchestration*.
+
+This high-level diagram easily illustrates the complexity involved.
+
+> Figure 1. A sneak peak at the map of ML development and product path
+![A sneak peak at the map of ML development and product path](image.png)
```

Comment:
Is there an easy way to have this image open up in a new tab at full size so i can zoom in and see all the details?